packages/forms/resources/views/components/repeater/simple.blade.php

```diff
@@ -50,8 +59,14 @@ class="fi-fo-repeater-item simple flex justify-start gap-x-3"
                                 {{ $item }}
                             </div>
 
-                            @if ($isReorderableWithDragAndDrop || $isReorderableWithButtons || $isCloneable || $isDeletable)
+                            @if ($isReorderableWithDragAndDrop || $isReorderableWithButtons || $isCloneable || $isDeletable || filled($visibleExtraItemActions))
```

Comment:
Empty array is falsy, so no need to wrap it in `filled()`, right?

Reply:
This is right.